### ISSUE TITLE
overriding colors doesn't work.

### ISSUE THREAD

**Issue by notaduck** [NLKNguyen/papercolor-theme#107]
I have tried to change the background for the `default.light` by adding the folowing lines to my `.vimrc` file
```
let g:PaperColor_Theme_Options = {
  \   'theme': {
  \     'default.light': { 
  \       'override' : {
  \         'color00' : ['#dfddd5', '232']
  \       }
  \     }
  \   }
  \ }
```
But the background color doesn't change at all?

**Comment by NLKNguyen** [comment]
Did you put it before the line `color PaperColor` ?

**Comment by notaduck** [comment]
nope, I placed it below. in this order:

```
colorscheme PaperColor 
set bg=light

let g:PaperColor_Theme_Options = {
  \   'theme': {
  \     'default.light': { 
  \       'override' : {
  \         'color00' : ['#dfddd5', '232']
  \       }
  \     }
  \   }
  \ }

```

**Comment by notaduck** [comment]
I tried to change `colorscheme` to `color` without succes.

**Comment by NLKNguyen** [comment]
That's strange. I can see the background changed. I tested with GVim 8. What Vim are you using?

![image](https://user-images.githubusercontent.com/4667129/36282159-267ce724-1255-11e8-81c4-3167a4e3bb4d.png)

```
let g:PaperColor_Theme_Options = {
  \   'theme': {
  \     'default.light': { 
  \       'override' : {
  \         'color00' : ['#dfddd5', '232']
  \       }
  \     }
  \   }
  \ }

color PaperColor
set bg=light

```

**Comment by notaduck** [comment]
wow, this is really strange. I copied your settings above, and instead of the white background it became black..... here is the full [´.vimrc´](https://pastebin.com/Wrh1QWTS
![screenshot 44](https://user-images.githubusercontent.com/9454064/36297618-07c3527e-12f4-11e8-9ebc-6346e2ccce0d.jpg)

**Comment by notaduck** [comment]
@NLKNguyen what terminal do you use? I might think it is my therminal which is causing the error since yours seem to work fine and gvim seems to do the job as well imgur.com/a/0kkik

**Comment by NLKNguyen** [comment]
Yeah, I think I understand the issue now. That seems to me that it actually behaves correctly and not an error. Maybe I didn't state it clear enough in the instruction. 

So basically, if your Vim has true color (GUI or NVim with true color on), then the hex option will be used, in this case a shade of gray (that is what I saw in my GVim).  Now if you are in a 256 terminal, the 256 option will be used, which is that black color. I've just tried in Gnome and it is exactly your screenshot.

The hex color and 256 color should be similar. If you don't want to manage both, then just provide one, and make the other empty string `''`. Remember if you provide only hex, then in the 256 environment, the converter will try its best to find the closest 256 color, but rarely can it be exact when you go from true color to 256. If you provide 256 color, the hex color will be exact. PaperColor default theme uses only 256 color, so it looks identically in true color environment.

**Comment by flocc** [comment]
I encountered same problem. I solved this by manually copying theme into .vim/colors (instead of using vim-plug). Now override works.
